.github/workflows/release-cli.yml

```diff
@@ -92,30 +92,46 @@ jobs:
   signed-sbom:
     runs-on: ubuntu-22.04
     steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Setup Go environment
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
         with:
-          ref: ${{ inputs.ref || github.head_ref }}
-      - name: Download syft & grype
+          go-version: "1.19.5"
+      - name: Download Syft & Grype
+        shell: bash
         run: |
-          SYFT_VERSION=0.62.1
-          GRYPE_VERSION=0.53.1
+          SYFT_VERSION=0.66.1
+          GRYPE_VERSION=0.55.0
           curl -LO https://github.com/anchore/syft/releases/download/v${SYFT_VERSION}/syft_${SYFT_VERSION}_linux_amd64.tar.gz
           tar -xzf syft_${SYFT_VERSION}_linux_amd64.tar.gz
```

Comment:
Just to be safe, we could run the download from `/tmp` or manually delete the `tar.gz` and its contents after extracting them.

Reply:
Since the step is now before the Checkout action it should be good (PWD is some temp dir).

Reply:
Done, working-directory is now explicitly `/tmp`.